### PR TITLE
add 6 blocks with stars

### PR DESCRIPTION
**Pull request by oliaShymanska**
- [DEMO LINK](https://oliaShymanska.github.io/layout_stars/)
- [TEST REPORT LINK](https://oliaShymanska.github.io/layout_stars/report/html_report/)

- [x] Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
- [x] Each BEM block has its own separate file
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/styles/block/stars.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(../../images/star-active.svg);
+}
```
Comment (volodymyr-soltys97) [inline src/styles/block/stars.css:25]: These selectors must be separated by commas

File: src/styles/block/stars.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+
+  /* background-size: contain; */
```
Comment (volodymyr-soltys97) [inline src/styles/block/stars.css:31]: Remove comments everywhere

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
You need to remove margin right for the last star
image

File: src/styles/block/stars.css (hunk 2 above)
Comment (oliaShymanska) [reply src/styles/block/stars.css:31]: When I add the property ".stars__star:last-child { margin-right: 0;}"  the test fails (npm t).

**Pull request closed without merging**